src/components/screens/wallet/send/form/index.js

```diff
@@ -1,20 +1,14 @@
 /* istanbul ignore file */
 import { connect } from 'react-redux';
 import { withTranslation } from 'react-i18next';
-import { dynamicFeesRetrieved } from '../../../../../actions/service';
 import { getActiveTokenAccount } from '../../../../../utils/account';
 import Form from './form';
 
 const mapStateToProps = state => ({
   account: getActiveTokenAccount(state),
   bookmarks: state.bookmarks,
   token: state.settings.token && state.settings.token.active,
-  dynamicFees: state.service.dynamicFees,
   networkConfig: state.network,
 });
 
-const mapDispatchToProps = {
-  dynamicFeesRetrieved,
-};
-
-export default connect(mapStateToProps, mapDispatchToProps)(withTranslation()(Form));
+export default connect(mapStateToProps)(withTranslation()(Form));
```

Comment:
This would require a lot of changes to the tests. As now the approach is to mount the component without the HOCs. 

There are several different ways how to handle the tests https://stackoverflow.com/questions/56827300/how-to-test-a-component-using-react-redux-hooks

IMO We should have an STM to chose one approach and update our testing guide accordingly. IMO there's a lot to do and thus I suggest this topic is outside the scope of this PR.

Reply:
Good point. We can do this later.